kover.gradle

```diff
@@ -0,0 +1,62 @@
+apply plugin: 'kover'
+
+project.ext {
+    ignoreModuleList = [
+            'com.amplifyframework.testutils.*',
```

Comment:
I think it would be better to just disable kover entirely for these modules as this is still generating an (empty) coverage file for these modules. I was able to do it by adding this to the end of the file:

```groovy
tasks.withType(Test) {
    kover {
       isDisabled = project.name.contains("test")
    }
}
```